components/remote_receiver.rst

```diff
@@ -55,18 +55,22 @@ Configuration variables:
   - **sony**: Decode and dump Sony infrared codes.
   - **toshiba_ac**: Decode and dump Toshiba AC infrared codes.
 
-- **tolerance** (*Optional*, int): The percentage that the remote signal lengths can deviate in the
-  decoding process. Defaults to ``25%``.
+- **tolerance** (*Optional*, int or :ref:`config-time`): The percentage or time that the remote signal lengths can
+  deviate in the decoding process. Allowed values are in range ``0..4294967295%`` or ``0..4294967295us``. Defaults to
```

Comment:
When I first read this I thought the double-decimal thing was a typo. Let's just write it out.
```suggestion
  deviate in the decoding process. Allowed values are in range ``0`` to ``4294967295%`` or ``0`` to ``4294967295us``. Defaults to
```